### PR TITLE
AbortSubtask RPC

### PR DESCRIPTION
**Pull request by Wiezzel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,10 @@ For requestor the app should implement a long running RPC service which implemen
   - Shouldn't take much time (preferably less than a minute for medium range machines).
 - `AbortTask`
   - Takes one argument: `task_id`.
-  - Will be called when the task is aborted by the user. Should stop all running subtask verifications for this task and perform any other necessary cleanup.
+  - Will be called when the task is aborted by the user or timed out. Should stop all running subtask verifications for this task and perform any other necessary cleanup.
+- `AbortSubtask`
+  - Takes two arguments: `task_id` and `subtask_ids`.
```
Comment (maaktweluit) [inline README.md]:
```suggestion
  - Takes two arguments: `task_id` and `subtask_id`.
```

File: README.md [hunk 2]
```diff
@@ -60,7 +60,10 @@ For requestor the app should implement a long running RPC service which implemen
...
+  - Will be called when the subtask is aborted by the user timed out. Should stop verification of the subtask (if it's running) and perform any other necessary cleanup.
```
Comment (maaktweluit) [inline README.md]:
```suggestion
  - Will be called when the subtask is aborted by the user or timed out. Should stop verification of the subtask (if it's running) and perform any other necessary cleanup.
```

Comment (maaktweluit) [inline README.md]: Should it stop the verification? then there is a chance the subtask will get assigned again

File: python/requirements-build.txt [hunk 3]
```diff
@@ -1 +1,2 @@
 grpcio-tools
+dataclasses
```
Comment (maaktweluit) [inline python/requirements-build.txt]: why is this needed? i do not see it imported anywhere, am i missing something?

File: README.md (hunk 2 above)
Comment (Wiezzel) [reply README.md]: I thought that's what we agreed on.

Comment (maaktweluit) [reply README.md]: yes, we agreed on not calling the app_client from RTM when the status is verifying.

Just when reading it i thought about `get_next_subtask` that will get a race condition

File: python/requirements-build.txt (hunk 3 above)
Comment (Wiezzel) [reply python/requirements-build.txt]: It's used in `structs`.  Without installing it `gen_constants` would fail.
